parts/k8s/kuberneteswinagentresourcesvmss.t

```diff
@@ -82,7 +82,7 @@
           ]
         },
         "osProfile": {
-          "computerNamePrefix": "[concat(substring(parameters('nameSuffix'), 0, 5), 'acs')]",
```

Comment:
Should we do this concatenation in-code as opposed to having ARM do it? @tariq1890 thoughts?

Reply:
Yup! We could have a template function call instead of the ARM stuff here and that template call can do the same thing what's being done right. nameSuffix is already accessible via public methods.

Reply:
@jackfrancis my change actually removes this line containing the concatenation and replaces it with a single variable I'm confused what concatenation you're referring to

Reply:
Oh jeez color blindness